blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,76 @@
+:root {
+  --v2-feature-carousel-card-width: 150px;
+  --v2-feature-carousel-card-height: 200px;
+  --v2-feature-carousel-icon-size: 48px;
+  --v2-feature-carousel-svg-size: 24px;
+  --v2-feature-carousel-border-radius: var(--border-radius);
+}
+
+.v2-feature-carousel-container {
+  background-color: var(--c-grey-50);
+}
+
+.v2-feature-carousel__cards-list {
+  display: flex;
+  gap: 16px;
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  flex-flow: row nowrap;
+}
+
+.v2-feature-carousel__card {
+  display: flex;
+  flex: 0 0 auto;
+  width: var(--v2-feature-carousel-card-width);
+  height: var(--v2-feature-carousel-card-height);
+  padding: 24px;
```

Comment:
You can create a variable for the padding, and change it to `32px 24px` in desktop ;)